models/DataObject/Concrete.php

```diff
@@ -647,13 +646,23 @@ public static function __callStatic($method, $arguments)
         }
         $propertyName = implode('', preg_grep('/^' . preg_quote($propertyName, '/') . '$/i', $fieldnames));
```

Comment:
**getByFieldName** not working for localized fields. It throws error "Call to undefined static method getBy...."

Reply:
Fixed.